tests/aws-cpp-sdk-core-tests/endpoint/endpointParamTest.cpp

```diff
@@ -0,0 +1,93 @@
+
```

Comment:
I guess you created a separate package for this test suite and forgot to delete this file.
Please remove this file if so.

Reply:
ack